leader-election-impl/src/main/java/com/palantir/paxos/PaxosStateLogMigrator.java

```diff
@@ -83,5 +91,25 @@ private long lowestSequenceToMigrate() {
         PaxosStateLog<V> destinationLog();
         Persistable.Hydrator<V> hydrator();
         SqlitePaxosStateLogMigrationState migrationState();
+        OptionalLong migrateFrom();
```

Comment:
discuss: either make this non-`Optional`, or add a safety buffer in the ranges above before doing the actual migration